core/Azure.Mcp.Core/src/Areas/Server/Commands/ServiceStartCommand.cs

```diff
@@ -148,6 +149,9 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
         {
             using var host = CreateHost(options);
 
+            var httpClientService = host.Services.GetRequiredService<IHttpClientService>();
+            BaseAzureService.InitializeHttpClientTransport(httpClientService);
```

Comment:
Can you please detail the goal of this PR and the changes you want to make? If I understand the motivation, I can help us make sure we implement something safe for local and remote MCP server scenarios.